scripts/docker_up.sh

```diff
@@ -15,16 +12,10 @@ set -e
 # Kafka needs to know our ip address so that it can advertise valid
 # connnection details
 
-if [ -z ${IP_IFACE} ]
-then
-  echo Detecting active network interface
-  IP_IFACE=$(ifconfig | ./scripts/active_ifaces.sh | head -n 1 | cut -d ':' -f1)
-fi
-
-export DOCKER_IP=$(ifconfig ${IP_IFACE} | grep 'inet ' | awk '{print $2}' | cut -d ':' -f2 | grep -v '127.0.0.1')
+export DOCKER_IP=$(./scripts/docker_ip.sh)
 
 # for debugging purposes
-echo Detected active network interface ${IP_IFACE} with ip ${DOCKER_IP}
+echo Detected active ip address ${DOCKER_IP}
```

Comment:
nice. I believe this makes `docker_up_ubuntu.sh` useless and that you can remove it in this PR